src/domains/Townhall/TownhallChat/TownhallChatUser.jsx

```diff
@@ -4,11 +4,11 @@ import Container from '@material-ui/core/Container';
 import Grid from '@material-ui/core/Grid';
```

Comment:
I don't think this file should have been modified, probs a vscode thing.

Reply:
Yea, vscode be like that